src/stats/skills.js

```diff
@@ -12,10 +12,7 @@ async function getLevels(userProfile, profileMembers, hypixelProfile, levelCaps)
     }, 0);
 
     Object.assign(skillLevels, {
-      taming: getLevelByXp(SKILL.SKILL_TAMING, {
-        skill: "taming",
-        cap: Math.max(hypixelProfile.achievements.skyblock_domesticator, 50),
-      }),
+      taming: getLevelByXp(SKILL.SKILL_TAMING, {skill: "taming", cap: levelCaps.taming }),
```

Comment:
that's gonna just remove cap completely?

Reply:
> that's gonna just remove cap completely?

No just moved the cap for the "levelCaps" for consistency, same way farming / carpentry is done (Line 79)